pylops/utils/decorators.py

```diff
@@ -0,0 +1,76 @@
+from functools import wraps
+from typing import Callable, Optional
+
+
+def reshaped(
+    func: Optional[Callable] = None,
```

Comment:
mmh have we started introducing typing annotations?

I wonder if it is best to wait and then do this in one unique 'boring' PR so we make sure everything is done consistently across the library?

Reply:
Fair enough! Removed them.